packages/common/src/openapi/schemas/Dashboard.json

```diff
@@ -34,6 +34,9 @@
         },
         "updatedByUser": {
             "$ref": "../Schemas/UpdatedByUser.json"
+        },
+        "pinnedListUuid": {
```

Comment:
are we still updating openapi ?

Reply:
If we don't update the local env doesn't return that property to the FE 😢